contracts/CTTreasury.sol

```diff
@@ -418,6 +442,28 @@ contract CTTreasury is SolidDaoManaged, ERC1155Receiver {
     ************************************************************
     */
 
+    /**
+    * @notice function where the Governor sets the DAO liquidity fee
+    * @dev only governor can call this function
+    * @return true if everything goes well
+    * @param _daoLiquidityFee uint256 to store the DAO Liquidity Fee
+    */
+    function setDAOLiquidityFee(uint256 _daoLiquidityFee) external onlyGovernor returns(bool) {
+        daoLiquidityFee = _daoLiquidityFee;
+        return true;
+    }
+
+    /**
+    * @notice function where the Governor sets the DAO Smart Contract address
+    * @dev only governor can call this function
+    * @return true if everything goes well
+    * @param _daoAddress address to store the DAO Smart Contract address
+    */
+    function setDAOLiquidityFee(address _daoAddress) external onlyGovernor returns(bool) {
```

Comment:
Probably `setDAOAddress`. What is the DAO address? An entry point for the whole application or what?

Reply:
I wasn't clear, this method should have name "setDAOAddress", because "setDAOLiquidityFee" is already declared above.

Reply:
Thanks for the heads up. Function renamed.